wasm.rkt

```diff
@@ -69,6 +80,80 @@
                                                   0
                                                   1))])
 
+;;;;;;;;; TABLE
+
+;; add table
+(define-metafunction WASM-eval
+  table-add : s tab -> s
+  [(table-add (s-func s-table s-mem) (table c_1 table-type )) (s-func (table-type c_1 (const 0) (const 0) $nopointer s-table) s-mem)])
+
+;;update table offset
+(define-metafunction WASM-eval
+  table-update-offset : s-func s-mem s-table v -> s
+  [(table-update-offset s-func s-mem (table-type c_size v_offset v_index i_1 s-table)  v_new)
+   (s-func  (table-type c_size v_new v_index i_1 mt-s-table) s-mem)])
+
+;;set table offset
+(define-metafunction WASM-eval
+  table-set-offset : s v -> s
+  [(table-set-offset (s-func s-table s-mem)  v_offset) (table-update-offset s-func s-mem s-table v_offset)])
```

Comment:
can we condense `table-set-offset` and `table-update-offset` into one metafunction?

Reply:
since table offset will no longer be necessary, the two meta-functions have been expunged